tests/wp-env/config/setup.sh

```diff
@@ -1,56 +1,10 @@
 #!/bin/bash
 set -eox pipefail
 
-echo "=== SETUP.SH DEBUG ==="
```

Comment:
Do we need to make this change? These change might be a bit sensitive, because I have made changes in the same files.

Reply:
I will roll back as they are failing anyway, but locally they were ok